pombola/south_africa/templates/south_africa/mp_attendance.html

```diff
@@ -5,6 +5,9 @@
 
 {% block js_end_of_body %}
   {{ block.super }}
+  <script type="text/javascript">
+    var POSITION = '{{ position }}';
```

Comment:
It doesn't actually matter in this case, since you know that `position` won't have unusual characters in it, but in case someone changes the view code or copy-and-pastes this to somewhere where it isn't safe, I'd make it: `position|escapejs`.